eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -284,6 +284,9 @@ services:
         class: "%ezpublish.image_alias.variation_purger.io.class%"
         arguments:
             - "@ezpublish.fieldType.ezimage.io_service"
+            - "@ezpublish.cache_pool"
+            - '@Ibexa\Core\Persistence\Cache\Identifier\CacheIdentifierGeneratorInterface'
+            - "@ezpublish.image_alias.imagine.cache.alias_generator_decorator"
             - "@ezpublish.image_alias.variation_path_generator.alias_directory"
```

Comment:
CS: `"/'` usage mix. The correct one is the single quote, but I won't object if you keep double quotes for the diff simplicity.

Reply:
`"` won't work in this case because of `does not contain valid YAML: Found unknown escape character "\C" at line 288 `.

Reply:
@barw4 it would, you'd just need to escape it proplerly. In this case though change the whole definition to use single quotes please. This is the direction we will go globally, once we'll have some time :tm:

Reply:
Resolved in https://github.com/ezsystems/ezpublish-kernel/pull/3130/commits/185b8ae638fb628ef62a4f48314df6f39032c947